packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -132,6 +136,7 @@ export const outOfOfficeCreateOrUpdate = async ({ ctx, input }: TBookingRedirect
   const createdOrUpdatedOutOfOffice = await prisma.outOfOfficeEntry.upsert({
     where: {
       uuid: input.uuid ?? "",
+      userId: ctx.user.id,
```

Comment:
We don't need this because if the user is unauthorized, he/she cannot access the platform.

Reply:
An unauthorised user (not unauthenticated) can delete another user's OOO entry and also mark themselves as the "redirected to" contact during OOO days. In the below video, member1 can delete the ooo entry of owner1 and also mark himself as the "redirected to" contact during the ooo days.

https://www.loom.com/share/7e560b67f31841708690bd827e89edaf?sid=57e1bef6-0603-40aa-b9ab-41901f326aa9